src/WidgetMessaging.js

```diff
@@ -87,6 +87,19 @@ export default class WidgetMessaging {
         });
     }
 
+    /**
+     * Tells the widget that it should terminate now.
+     * It is not necessarily called in all instances before the widget is removed,
+     * and the client may force termination with a timeout.
```

Comment:
We shouldn't need to clarify when or how this is called at this layer - all actions are already subject to timeouts, and how they're called is not relevant to the API shape.